consensus/consensus_tests/utils_test.go

```diff
@@ -104,11 +101,15 @@ func CreateTestConsensusPocketNodes(
 // Creates a pocket node where all the primary modules, exception for consensus, are mocked
 func CreateTestConsensusPocketNode(
 	t *testing.T,
-	cfg *genesis.Config,
-	genesisState *genesis.GenesisState,
+	cfg modules.Config,
+	genesisState modules.GenesisState,
 	testChannel modules.EventsChannel,
 ) *shared.Node {
-	consensusMod, err := consensus.Create(cfg, genesisState)
+	config, err := json.Marshal(cfg.Consensus)
+	require.NoError(t, err)
+	genesis, err := json.Marshal(genesisState.ConsensusGenesisState)
```

Comment:
s/genesis/consensusGenesis

Reply:
?

Reply:
It's linux-like `sed` structure often use in comments when asking to rename 

It implies: Replace `genesis` with `consensusGenesis`.

Reference to screenshot: https://www.grymoire.com/Unix/Sed.html
<img width="1396" alt="Screen Shot 2022-09-03 at 1 26 49 PM" src="https://user-images.githubusercontent.com/1892194/188281828-33f507ef-60f1-407e-86b1-c9b2dc7bd324.png">